debugger.md

```diff
@@ -38,10 +38,10 @@ You can start from scratch by creating an empty project with `dotnet new`. Begin
     dotnet new
     dotnet restore
 
-If you want a web project (ASP.NET project) use `dotnet new -t web`. For web projects, makes sure to run `bower install` before running so that they can restore assets.
+If you want a web project (ASP.NET project) use `dotnet new web`. For web projects, makes sure to run `bower install` before running so that they can restore assets.
 
 ##### 2: Open the directory in VS Code
-Go to `File->Open` and open the directory in Visual Studio Code. If this is the first time that the C# extension has been activated, it will now download additional platform-specific dependencies.
+Go to `File->Open Folder` and open the directory in Visual Studio Code. If this is the first time that the C# extension has been activated, it will now download additional platform-specific dependencies.
```

Comment:
```suggestion
Go to `File->Open Folder` (`File->Open` on macOS) and open the directory in Visual Studio Code. If this is the first time that the C# extension has been activated, it will now download additional platform-specific dependencies.
```